app/server/apiProxy.ts

```diff
@@ -92,12 +108,11 @@ export const proxyApiHandler = (
 
       res.status(intermediateResponse.status);
 
-      headersToForward.forEach((headerName: string) =>
-        res.header(
-          headerName,
-          intermediateResponse.headers.get(headerName) || undefined
-        )
+      res.header(
+        MDA_TEST_USER_HEADER,
+        intermediateResponse.headers.get(MDA_TEST_USER_HEADER) || undefined
```

Comment:
the only use case of the `headersToForward` feature was the test-user header, so we simplified (i.e. removed premature abstraction/generalisation)